### PR TITLE
inverseRobot solution

### PR DESCRIPTION
**Pull request by maksym-maliuk**

### CODE REVIEW

**Review by Lilu580 — COMMENTED** [review]
Gj

**Review by Stepan-TS — APPROVED** [review]

File: src/inverseRobot.js [hunk 1]
```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+  const key = Object.keys(robot);
+  const value = Object.values(robot);
```
Comment (Andr1yK) [inline src/inverseRobot.js:12]: array with keys named as a key.. something is wrong

**Review by Andr1yK — CHANGES_REQUESTED** [review]
now your code had a big algorithmic complexity, let's simplify it)

U can use just `for in` loop

File: src/inverseRobot.js [hunk 2]
```diff
@@ -7,7 +7,16 @@
...
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const key in robot) {
+    if (newRobot[robot[key]]) {
+      return null;
+    }
+    newRobot[robot[key]] = key;
```
Comment (Andr1yK) [inline src/inverseRobot.js:16]:
```suggestion
    if (newRobot[robot[key]]) {
      return null;
    }

    newRobot[robot[key]] = key;
```

**Review by Andr1yK — APPROVED** [review]

**Pull request closed without merging**